app/views/games/index.html.erb

```diff
@@ -1,10 +1,8 @@
 <h1>Welcome to RevelDevs Chess!</h1>
 <br />
-<h3><%= link_to "Create a Game", new_game_path(@game), class: 'btn btn-primary' %></h3>
-<h3><%= link_to "Join a Game", "#", class: 'btn btn-primary' %></h3>
+<h3><%= link_to "Create a Game", new_game_path, class: 'btn btn-primary' %></h3>
 
-
-<h3>Available Games</h3>
-<% @available_games.each do |f| %>
-<p>Game Name: <%= f.name%> | Total Players: <%= f.total_players%></p>
-<%end%>
\ No newline at end of file
+<% @games.each do |game| %>

```

Comment:
I think this still needs to be `@available_games.each`. We only want to display games that currently have 1 player (and are therefore "joinable").

Reply:
changed back